File: 백준/Bronze/30979. 유치원생 파댕이 돌보기/유치원생 파댕이 돌보기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int T = sc.nextInt();
        int N = sc.nextInt();

        int sum = 0;
        for (int i = 0; i < N; i++) {
            sum += sc.nextInt();
        }

        if (T <= sum) {
            System.out.println("Padaeng_i Happy");
        } else {
            System.out.println("Padaeng_i Cry");
        }
    }
}

File: 백준/Bronze/30979. 유치원생 파댕이 돌보기/유치원생 파댕이 돌보기.cs
int T = int.Parse(Console.ReadLine());
int N = int.Parse(Console.ReadLine());
int[] F = Console.ReadLine().Split().Select(int.Parse).ToArray();

if (T <= F.Sum())
{
    Console.WriteLine("Padaeng_i Happy");
}
else
{
    Console.WriteLine("Padaeng_i Cry");
}